lib/types/index.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+    bmp: require('./bmp'),
+    cur: require('./cur'),
+    dds: require('./dds'),
+    gif: require('./gif'),
+    ico: require('./ico'),
+    jpg: require('./jpg'),
+    png: require('./png'),
+    psd: require('./psd'),
+    svg: require('./svg'),
+    tiff: require('./tiff'),
+    webp: require('./webp')
+};
```

Comment:
missing newline at end of file, and the other files are using two spaces as indentation